src/containers/Mainnet/data.js

```diff
@@ -0,0 +1,60 @@
+export const mainnetIbcChannels = [
+    {
+      key: '1',
+      source_channel: 'channel-51',
+      destination_chain: 'Axelar',
+      destination_chainid: 'axelar-dojo-1',
+      destination_channel: 'channel-42',
+      known_operators: 'AUDIT.one, Cros-nest, Polkachu *looking for more operators*'
+    },
+    {
+      key: '2',
+      source_channel: 'channel-35',
+      destination_chain: 'Comdex',
+      destination_chainid: 'comdex-1',
+      destination_channel: 'channel-10',
+      known_operators: 'Architect Nodes, Cros-nest *looking for more operators*'
+    },
+    {
+      key: '3',
+      source_channel: 'channel-24',
+      destination_chain: 'Cosmos Hub',
+      destination_chainid: 'cosmoshub-4',
+      destination_channel: 'channel-190',
+      known_operators: 'AUDIT.one ⚡, Architect Nodes ⚡️, Cros-nest ⚡️, CryptoCrew ⚡, DSRV, Stakin'
+    },
+    {
+      key: '4',
+      source_channel: 'channel-68',
+      destination_chain: 'Crescent',
+      destination_chainid: 'crescent-1',
+      destination_channel: 'channel-30',
+      known_operators: 'Simply Staking *looking for more operators*'
+    },
+    {
+      key: '5',
+      source_channel: 'channel-38',
+      destination_chain: 'Gravity Bridge',
+      destination_chainid: 'gravity-bridge-3',
+      destination_channel: 'channel-24',
+      known_operators: 'AUDIT.one, bitszn, Cosmonaut Stakes, CryptoCrew, Stakin'
+    },
+    {
+      key: '6',
+      source_channel: 'channel-68',
+      destination_chain: 'Crescent',
+      destination_chainid: 'crescent-1',
+      destination_channel: 'channel-30',
+      known_operators: 'Simply Staking *looking for more operators*'
+    },
+    {
+      key: '7',
+      source_channel: 'channel-38',
+      destination_chain: 'Gravity Bridge',
+      destination_chainid: 'gravity-bridge-3',
+      destination_channel: 'channel-24',
+      known_operators: 'AUDIT.one, bitszn, Cosmonaut Stakes, CryptoCrew, Stakin'
```

Comment:
Same here. Don’t think audit one is relaying

Reply:
Need proper data for same. Data taken from designs